standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestMetastoreScheduledQueries.java

```diff
@@ -357,6 +360,17 @@ public void testPoll() throws Exception {
     assertFalse(pollResult.isSetQuery());
   }
 
+  private String generateLongErrorMessage() {
+    StringBuffer sb=new StringBuffer();
```

Comment:
This is a test, so not super important, but this should be a `StringBuilder` and please fix the checkstyle issue (no space between the `=`.  Thanks.